msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/TestCase1561080.java

```diff
@@ -0,0 +1,208 @@
+import com.microsoft.identity.labapi.utilities.jwt.JWTParserFactory;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.concurrent.TimeUnit;
+
+// [WPJ] Get device state
+// https://identitydivision.visualstudio.com/Engineering/_testPlans/define?planId=1905195&suiteId=1905204
+@SupportedBrokers(brokers = {BrokerMicrosoftAuthenticator.class})
+public class TestCase1561080 extends AbstractMsalBrokerTest {
+
+    final String MY_ACCOUNT_MICROSOFT_URL = "https://myaccount.microsoft.com/";
+
+    @Test
+    public void test_1561080() throws Throwable {
+        final String username = mLabAccount.getUsername();
+        final String password = mLabAccount.getPassword();
+
+        final BrokerMicrosoftAuthenticator brokerMicrosoftAuthenticator = (BrokerMicrosoftAuthenticator) mBroker;
+
+        //perform device registration
+        brokerMicrosoftAuthenticator.performDeviceRegistration(username, password);
+
+        BrowserChrome chrome  = new BrowserChrome();
+        chrome.clear();
+        chrome.launch();
+        chrome.handleFirstRun();
+
+        chrome.navigateTo(MY_ACCOUNT_MICROSOFT_URL);
+
+        final AadLoginComponentHandler aadLoginComponentHandler = new AadLoginComponentHandler();
+        aadLoginComponentHandler.handleEmailField(username);
+        aadLoginComponentHandler.handlePasswordField(password);
+
+        UiAutomatorUtils.handleButtonClick("com.android.chrome:id/button_secondary");
+
+        UiAutomatorUtils.handleButtonClickForObjectWithText("MANAGE DEVICES");
+
+        UiAutomatorUtils.handleButtonClickForObjectWithText("Collapsible item");
+
+        UiObject disable_lost_device_btn1 = UiAutomatorUtils.obtainUiObjectWithText("Disable lost device");
+        disable_lost_device_btn1.click();
+
+        UiObject disable_lost_device_btn2 = UiAutomatorUtils.obtainUiObjectWithText("Disable lost device");
+        disable_lost_device_btn2.click();
+
+        //install prod brokerhost app.
+        BrokerHost brokerHost = new BrokerHost(BrokerHost.BROKER_HOST_APK_RC);
+        if(brokerHost.isInstalled()){
+            brokerHost.uninstall();
+        }
+
+        brokerHost.install();
+
+        brokerHost.launch();
+        //run get device state
+        String deviceStateResponse = brokerHost.getDeviceState();
+
+        Assert.assertEquals("false",deviceStateResponse);
+
+        //Get the device ID direct from the brokerHost.
+        String deviceId = brokerHost.obtainDeviceId();
+
+        //launch msal test app.
+        final MsalSdk msalSdk = new MsalSdk();
+
+        final MsalAuthTestParams authTestParams_firstTry = MsalAuthTestParams.builder()
+                .activity(mActivity)
+                .loginHint(username)
+                .scopes(Arrays.asList(mScopes))
+                .promptParameter(Prompt.LOGIN)
+                .msalConfigResourceId(getConfigFileResourceId())
+                .build();
+
+        //AT interactive acquisition. - It should fail
```

Comment:
Why does it fail, what error/exception?

Reply:
I think this is because device is disabled